rust/crd/src/lib.rs

```diff
@@ -1301,6 +1303,7 @@ mod tests {
                 read_only: None,
                 sub_path: None,
                 sub_path_expr: None,
+                recursive_read_only: None,
```

Comment:
We could default here for all "None" options as well?

Reply:
done https://github.com/stackabletech/spark-k8s-operator/pull/401/commits/9157ebe34443c4278849ff5415cabb28077be378